### PR TITLE
[Chore] Tweet card

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- Update tweet card design

## Type of change
- [x] Chore (non-breaking change which does not add visible functionality but improves code quality)

## Screenshots
Screenshot 2021-12-14 at 15 23 43

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  images: {
+    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS.split(", "),
```
Comment (kilemensi) [inline next.config.js:3]:
```suggestion
    domains: process.env.NEXT_PUBLIC_IMAGE_DOMAINS?.split(",")?.map((d)=>d.trim()),
```
What's with the space (" ")?

File: src/components/Tweets/index.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ import TweetCard from "@/twoopstracker/components/TweetCard";
 const Tweets = ({ tweets, ...props }) => {
   const classes = useStyles(props);
 
+  console.log(tweets);
+
```
Comment (kilemensi) [inline src/components/Tweets/index.js:14]:
```suggestion
```

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

1. Images still don't load on Vercel (at least the preview link doesn't)
2. Truncate doesn't appear to handle html content
Screenshot from 2021-12-15 08-38-56

**Comment by KhadijaMahanga** [comment]
> 👍🏽
> 2. Truncate doesn't appear to handle html content
>    Screenshot from 2021-12-15 08-38-56

@kilemensi weird! Yes, I had trouble with this yesterday but found `react-truncate-markup` that handles html. Let me check it again

**Comment by kilemensi** [comment]
On mobile so it could be by design but the see more / see less toggle is now completely gone @KhadijaMahanga

**Review by gertie-sheshe — APPROVED** [review]
LGTM @KhadijaMahanga

**Review by KobbyMmo — APPROVED** [review]
Looks good to me @KhadijaMahanga

**Pull request merged**